pages/events/Working Group Meetings.mdx

```diff
@@ -0,0 +1 @@
+## Gray Foundation Annual Summit
```

Comment:
Update heading to ## Working Group Meetings

Elvira and I can go back and add more to this page.